graphium/data/datamodule.py

```diff
@@ -62,16 +64,13 @@
 from graphium.data.utils import graphium_package_path, found_size_mismatch
 from graphium.utils.arg_checker import check_arg_iterator
 from graphium.utils.hashing import get_md5_hash
-from graphium.data.smiles_transform import (
-    did_featurization_fail,
-    BatchingSmilesTransform,
-    smiles_to_unique_mol_ids,
-)
 from graphium.data.collate import graphium_collate_fn
 import graphium.data.dataset as Datasets
 from graphium.data.normalization import LabelNormalization
 from graphium.data.multilevel_utils import extract_labels
 
+import graphium_cpp
```

Comment:
The Python module implemented in C++ is currently named `graphium_cpp`.  I'm not sure how Python module namespacing works in this regard, but if you'd prefer a different name, it should be fairly straightforward to name it something different.

Reply:
I'm fine with the name `graphium_cpp`, my question is why isn't it `graphium.graphium_cpp`? When you do `pip install graphium`, it also installs another package named `graphium_cpp`? If so that's fine. I just don't want this import to be dependant on your cwd.